service-manual/making-software/security-testing.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: detailed-guidance
+title: Security testing
+subtitle: What is security testing and how to use it
+category: making-software
+type: guide
+audience:
+  primary: tech-archs, web-ops, qa
+  secondary: developers, performance-analysts
+phases:
+  - alpha
+  - beta
+  - live
+breadcrumbs:
+  -
+    title: Home
+    url: /service-manual
+  -
+    title: Making software
+    url: /service-manual/making-software
+---
+
+Security Testing intends to find flaws and vulnerabilities in an IT system, as well as the people and processes that surround the system. The practice includes:
+
+* formal security testing for live service accreditation, eg [penetration testing](https://www.gov.uk/service-manual/operations/penetration-testing.html), which aggressively attacks a system to check its defences, and [DDOS](http://en.wikipedia.org/wiki/Denial-of-service_attack) (Distributed Denial of Service) durability testing, which tries to make a service unavailable 
+* system feature testing, eg quality assurance testing of password functions
+
+
+## Finding the right skills
+
+People with the appropriate skills, tools and accreditation can carry out security testing. They can be developers, security specialists, software testers or quality analysts. Specialist suppliers offering security testing services can be found on the Government [Digital Marketplace](https://www.gov.uk/digital-marketplace).
+ 
+The test and quality analyst team must give appropriate coverage to internal and cyber security features on areas such as user administration, password functionality, role based access control and auditing. This testing of security features is different to formal security testing, which may be out of scope for the quality assurance team.
+
+
+## Distinction between 'formal security testing' and typical assurance testing of security features:
```

Comment:
It feels like the distinction here may be more about what it's appropriate for internal vs. external people to be testing, which will follow from a set of considerations such as who needs to be assured, skills on the team, the level of risk being considered and potentially formal requirements.

There are elements of penetration and DDOS testing that can be performed by staff on the project team, just as the security related assurance areas could be tested by a third-party.